openedx.yaml

```diff
@@ -0,0 +1,8 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: a11y
+oeps: {}
+openedx-release: {ref: master}
+owner: edx/edx-accessibility
+track-pulls: true
```

Comment:
This specifies whether the repo should show up in various PR reports that the opensource team does.